### PR TITLE
[IMP] hr_employee_firstname: rename hooks file

### PR DESCRIPTION
**Pull request by andreagidaltig**
Apply guidelines on name of hooks file.

### CODE REVIEW

**Comment by andreagidaltig** [comment]
Hi @luisg123v could you review this please?

**Comment by luisg123v** [comment]
Remove the maintainers part from description.

**Comment by andreagidaltig** [comment]
I just updated this, could you review it, please? @luisg123v

**Review by luisg123v — APPROVED** [review]
LGTM :+1: 

@moylop260 could you review/merge, please?

File: hr_employee_firstname/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "Fekete Mihai (Forest and Biomass Services Romania), "
     "Onestein, "
     "Odoo Community Association (OCA)",
-    "maintainer": "Savoir-faire Linux",
+    "maintainers": ["Savoir-faire Linux", "luisg123v"],
```
Comment (Saran440) [inline hr_employee_firstname/__manifest__.py:11]:
```suggestion
    "maintainers": ["Savoir-faire Linux"],
```

please add your contributor in file `readme/CONTRIBUTORS.rst`

Comment (luisg123v) [reply hr_employee_firstname/__manifest__.py:11]: @Saran440 why is that?

Maintainer and contributors are not the same.

Comment (moylop260) [reply hr_employee_firstname/__manifest__.py:11]: The change is so small that it does not need to be added to CONTRIBUTORS

But it is good to be maintainers in order to be notified of changes

**Review by moylop260 — APPROVED** [review]
👍

File: hr_employee_firstname/__manifest__.py (hunk 1 above)
Comment (Saran440) [reply hr_employee_firstname/__manifest__.py:11]: oh, sorry i misunderstand that it author field.

**Review by Saran440 — APPROVED** [review]

**Pull request merged**